packages/@react-spectrum/button/test/Button.test.js

```diff
@@ -217,4 +220,43 @@ describe('Button', function () {
     let button = getByRole('button');
     expect(document.activeElement).toBe(button);
   });
+
+
+  it('submit in form using space', function () {
```

Comment:
why aren't these tests in <Form /> tests instead of <Button />
Should there be a test where pressing enter on a checkbox or other appropriate element of the form submits the form?

Reply:
It's a bit weird, but my thinking was I was enabling Button functionality, not Form, so into Button tests they go, and they just need Form to help verify